scripts/validate-markdown-values

```diff
@@ -3,12 +3,11 @@
 set -e -o pipefail -u
 
 self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
+root_dir=$(cd $PWD &>/dev/null; pwd -P)
 source "$self_dir"/utils.sh
-
 # Currently, just check whether earliest supported IntelliJ version in README.md
 # is up to date with the source of truth (Gradle config).
-
-expected=$(./gradlew -q resolveIntelliJVersions --version-key=earliestSupportedMajor)
+expected=$(grep -Po '(?<=earliestSupportedMajor=)[0-9.]*[0-9]' "$root_dir"/intellijVersions.properties)
 actual=$(grep -Po '(?<=minimum required version is )[0-9.]*[0-9]' README.md)
```

Comment:
While you're here, use `root_dir` here for consistency as well

Reply:
If wee need this `source "$self_dir"/utils.sh` ? because I don't see any use (?)

Reply:
`error` and `die` ☠️

Reply:
aa okey :) !